fly/commands/set_pipeline.go

```diff
@@ -52,6 +52,7 @@ func (command *SetPipelineCommand) Execute(args []string) error {
 	atcConfig := setpipelinehelpers.ATCConfig{
 		Team:             target.Team(),
 		PipelineName:     pipelineName,
+		TargetName:       Fly.Target,
```

Comment:
Storing both the `TargetName` and `Target` in the `ATCConfig` is a little gross, IMO. Alternatively, the `ATCConfig` could contain a complete `Target` and `ATCConfig.Set()` could unpack the name and URL from it. Perhaps there is a reason (earlier validation of the target?) this is not desirable, though.

Reply:
To be honest I've been wanting to rid us of this whole `ATCConfig` abstraction anyway as it seems to just add unnecessary indirection, but I'm OK with throwing a tiny stick onto a burning tire fire for now. 😉